### ISSUE TITLE
Java generics with more then one parameter parsing bug

### ISSUE THREAD

**Issue by michacom** [spadgos/sublime-jsdocs#357]
Given function:
```java
public <V> V[] convertTo(Map<?,V> source, V[] destination) {
    return source.values().toArray(destination);
}
```

Expected result:
```java
/**
 * [convertTo description]
 * @param  source      [description]
 * @param  destination [description]
 * @return             [description]
 */
```
Actual result:
```java
/**
 * [convertTo description]
 * @param  Map<?,source,destination [description]
 * @return                          [description]
 */
```

But in C++ this works fine.

**Comment by michacom** [comment]
Sorry, but I'm completely lost with all these "branching", "requests Pull" ....
:disappointed:

**Comment by spadgos** [comment]
I found a neater way to fix it: 2a81cb4

Have a read of Github's docs if you'd like to know more about the general workflow. They have a lot of really good articles. https://help.github.com/articles/using-pull-requests/
